examples/benchmarks/tablebench/src/test/table.bench.ts

```diff
@@ -126,18 +126,43 @@ describe("Table", () => {
 				);
 			}
 
-			const dataBytes = measureEncodedLength(JSON.stringify(transposeTable(data)));
+			const rowMajorJsonBytes = measureEncodedLength(JSON.stringify(data));
+			const colMajorJsonBytes = measureEncodedLength(JSON.stringify(transposeTable(data)));
 			let summaryBytes: number;
 
 			// After each test, print the summary size information to the console.
 			afterEach(() => {
-				const ratio = summaryBytes / dataBytes;
-
 				// When using a logger, Mocha suppresses 'console.log()' by default.
 				// Writing directly to 'process.stdout' bypasses this suppression.
-				process.stdout.write(`Data: ${dataBytes} bytes\n`);
-				process.stdout.write(`Summary: ${summaryBytes} bytes\n`);
-				process.stdout.write(`Ratio: ${ratio}\n`);
+				process.stdout.write(`          Summary: ${summaryBytes} bytes\n`);
+				process.stdout.write(
+					`              vs row-major: ${(
+						summaryBytes / rowMajorJsonBytes
+					).toLocaleString(undefined, {
+						maximumFractionDigits: 2,
+						minimumFractionDigits: 2,
+					})}x\n`,
+				);
+				process.stdout.write(
+					`              vs col-major: ${(
+						summaryBytes / colMajorJsonBytes
+					).toLocaleString(undefined, {
+						maximumFractionDigits: 2,
+						minimumFractionDigits: 2,
+					})}x\n`,
+				);
+			});
```

Comment:
All the extra spaces above are to merge the indentation with mocha's output so the results look nice when I copy/paste into chats, emails, etc.